pkgs/servers/jigasi/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib, stdenv, fetchurl, dpkg, jdk11, nixosTests }:
+
+let
+  pname = "jigasi";
+  version = "1.1-231-gd792d0f";
+  arch = "amd64";
+  src = fetchurl {
+    url = "https://download.jitsi.org/stable/${pname}_${version}-1_${arch}.deb";
+    sha256 = "0riij1i3k33rgzs4gqjffvcrx18a1dmz825nzk85xpsh972z0a4n";
+  };
+in
+stdenv.mkDerivation {
+  inherit pname version src;
+
+  nativeBuildInputs = [ dpkg ];
+
+  dontBuild = true;
+
+  unpackCmd = "dpkg-deb -x $src debcontents";
+
+  installPhase = ''
+    runHook preInstall
+    substituteInPlace usr/share/${pname}/${pname}.sh \
+      --replace "exec java" "exec ${jdk11}/bin/java"
+
+    mkdir -p $out/{share,bin}
+    mv usr/share/${pname} $out/share/
+    mv etc $out/
+    ln -s $out/share/${pname}/${pname}.sh $out/bin/${pname}
```

Comment:
Please don't substitute pname in every possible location. This just makes the code harder to read.

Reply:
Alright, will change that.